### PR TITLE
All Expansion tab implemented

### LINKED ISSUE #3: Make an "All" tab in user mounts tab
**Is your feature request related to a problem? Please describe.**
- It is a hassle to have to switch between multiple tabs when updating the user mounts record for a single user.
- Updating a single user's user mounts record require multiple network calls.

**Describe the solution you'd like**
- Make an "All" tab in user mounts tab.

**Describe alternatives you've considered**
- N/A.

**Additional context**
- N/A.

### PR DESCRIPTION
**Pull request by EvanM27**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?
no ability to display all expansion userMounts on one table

Issue Number: [3](https://github.com/sofia819/mount_recorder_client/issues/3)

## What is the new behavior?
A new all tab displays all expansion userMounts on one table

## Other information
column spacing breaks on this table
edit modal does not function correctly displaying so many mounts
All Tab

### CODE REVIEW

**Comment by sofia819** [comment]
Can you make it so that the `All` tab isn't the second tab? I am thinking it should be the left most tab. You can give it an index of -1 instead.

**Review by sofia819 — CHANGES_REQUESTED** [review]
Can you make it so that the All tab isn't the second tab? I am thinking it should be the left most tab. You can give it an index of -1 instead.

File: src/utils/constants.js [hunk 1]
```diff
@@ -43,7 +43,9 @@ export const ADD_USER_BUTTON = "Add User";
 export const EXPANSION_COLUMN = "Expansion";
 
 export const EXPANSION_MAP = {
+  "-1": "all",
   0: "Misc.",
+  1: "All",
```
Comment (sofia819) [inline src/utils/constants.js]: Remove this

File: src/utils/constants.js [hunk 2]
```diff
@@ -43,7 +43,9 @@ export const ADD_USER_BUTTON = "Add User";
 export const EXPANSION_COLUMN = "Expansion";
 
 export const EXPANSION_MAP = {
+  "-1": "all",
```
Comment (sofia819) [inline src/utils/constants.js]: Uppercase `All`

**Review by sofia819 — APPROVED** [review]
Minor comments but LGTM

File: src/utils/constants.js (hunk 2 above)
Comment (EvanM27) [reply src/utils/constants.js]: Resolved

**Pull request merged**